src/ExposureHistory/ExposureDetail.tsx

```diff
@@ -22,10 +23,26 @@ const ExposureDetail: FunctionComponent = () => {
   useStatusBarEffect("light-content")
   const { t } = useTranslation()
 
+  const [connectivity, setConnectivity] = useState<boolean | null | undefined>(
```

Comment:
```suggestion
  const [isConnected, setIsConnected] = useState<boolean | null | undefined>(
```

Thoughts on this naming scheme to stay consistent with the rest of the codebase?

Reply:
Good catch!